v2/api/storage/v1api20220901storage/structure.txt

```diff
@@ -0,0 +1,187 @@
+// Code generated by azure-service-operator-codegen. DO NOT EDIT.
+github.com/Azure/azure-service-operator/v2/api/storage/v1api20220901storage
+├── APIVersion: Enum (1 value)
+│   └── "2022-09-01"
+├── StorageAccountsFileService: Resource
+│   ├── Owner: github.com/Azure/azure-service-operator/v2/api/storage/v1api20220901.StorageAccount
+│   ├── Spec: Object (6 properties)
+│   │   ├── Cors: *Object (2 properties)
+│   │   │   ├── CorsRules: Object (6 properties)[]
+│   │   │   │   ├── AllowedHeaders: string[]
+│   │   │   │   ├── AllowedMethods: string[]
+│   │   │   │   ├── AllowedOrigins: string[]
+│   │   │   │   ├── ExposedHeaders: string[]
+│   │   │   │   ├── MaxAgeInSeconds: *int
+│   │   │   │   └── PropertyBag: genruntime.PropertyBag
+│   │   │   └── PropertyBag: genruntime.PropertyBag
+│   │   ├── OriginalVersion: string
+│   │   ├── Owner: *genruntime.KnownResourceReference
+│   │   ├── PropertyBag: genruntime.PropertyBag
+│   │   ├── ProtocolSettings: *Object (2 properties)
+│   │   │   ├── PropertyBag: genruntime.PropertyBag
+│   │   │   └── Smb: *Object (6 properties)
+│   │   │       ├── AuthenticationMethods: *string
+│   │   │       ├── ChannelEncryption: *string
```

Comment:
Not sure if this one should be a secretRef?

Reply:
Not sure either -- look up an example where it's used?